wiki/Rules/Explicit_Content/id.md

```diff
@@ -0,0 +1,32 @@
+# Konten eksplisit
+
+![Tampilan beatmap yang mengandung konten eksplisit](img/explicit-tag.jpg "Contoh beatmap dengan ditandai dengan label 'eksplisit'.")
```

Comment:
```suggestion
![Tampilan beatmap yang mengandung konten eksplisit](img/explicit-tag.jpg "Contoh beatmap yang ditandai dengan label 'eksplisit'.")
```

Reply:
it doesn't feel natural using ```dengan``` when the next word after it is going to use ```dengan``` aswell.

Reply:
ya this is an oversight lol \:P

tyty